app/views/cases/attachments/new.html.slim

```diff
@@ -0,0 +1,37 @@
+- content_for :page_title do
+  = t('page_title.add_request_attachment_page', case_number: @case.number)
+
+= GovukElementsErrorsHelper.error_summary @case,
+  "#{pluralize(@case.errors.count, t('common.error'))} #{ t('common.summary_error')}", ""
+
+- content_for :heading
+  = t('cases.attachments.upload_request_files_heading')
+
+- content_for :sub_heading
+
+= link_to("Back", case_path(@case), class: 'govuk-back-link')
+
+span.visually-hidden
+  = t('common.case.header_case_number')
+  = @case.number
+
+= render partial: 'layouts/header'
+
+
+= form_for @case, as: :case_attachment, url: case_attachments_path(@case), method: :post do |f|
+  .form-group
+    .grid-row
+      .column-two-thirds
+        p.form-hint
+          = t('cases.attachments.upload_hint_copy')
+    .grid-row
+      section.case-main
+        = render partial: 'shared/dropzone_form',
+              locals: { s3_direct_post: @s3_direct_post,
+                        file_input_name: 'uploaded_request_files[]' }
+        .form-group
+          label.form_label
```

Comment:
Does this automatically add "for" attribute for the textarea?

Reply:
I don't know, let me check

Reply:
no 'for' is added, I added it

Reply:
Is it matching the ID of the textarea? I can't see the ID here so hard to tell.

Reply:
🤦‍♀️ ,   no, sorry! I corrected it "again", should it be right this time? 😳 
screenshot below
<img width="571" alt="Screenshot 2021-03-15 at 21 33 15" src="https://user-images.githubusercontent.com/26111671/111224428-5cb82680-85d6-11eb-94d7-80d6e0428679.png">